src/commands/create/contract.ts

```diff
@@ -209,8 +214,8 @@
           case BytecodeType.SampleERC721:
             contractType = 'HolographERC721'
             break
-          case BytecodeType.HolographDropsEditionsV1:
-            contractType = 'HolographDropsEditionsV1'
+          case BytecodeType.HolographDropERC721:
+            contractType = 'HolographDropERC721'
```

Comment:
would be it worth it to put `contractType = BytecodeType.HolographDropERC721` instead?